openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -470,7 +489,7 @@ def is_last_random_cohort(user_group):
     return len(random_cohorts) == 1 and random_cohorts[0].name == user_group.name
 
 
-def set_course_cohort_settings(course_key, **kwargs):
+def set_course_settings(course_key, **kwargs):
```

Comment:
I think we will want a different method for setting things relate to discussions vs. things related to cohorts.